hours_server.rb

```diff
@@ -58,6 +59,10 @@ class Server < Sinatra::Base
       end
       @student.add_lab_session(:time_in => Time.now)
 
+      if !REQUIRED_BUILD_DAYS.include?(DateTime.now.in_time_zone("America/Los_Angeles").strftime("%A"))
+        OptionalBuild.create(:date => DateTime.now.in_time_zone("America/Los_Angeles").strftime("%Y-%m-%d")) rescue nil
```

Comment:
Should we first check if the optional build already exists? Otherwise it'll try and fail to write for every sign-in, which seems sub-optimal.

Reply:
would probably be better convention to check first, will fix this at build today (or before)

Reply:
Fixed in 1ac702b, should be done now.